tox.ini

```diff
@@ -62,7 +62,7 @@ deps =
     twine: twine
 
     lint: pyflakes
-    lint: twistedchecker>=0.7.1
+    lint: git+https://github.com/twisted/twistedchecker@9796-rodrigc-no-pycodestylechecker
```

Comment:
I think that before merging this, we should release a new twistedchecker version and use a public released version here.

I think that if there is a new tag in the twistedchecker git repo, travis-ci will automatically publish a new package in pypi.

but it needs a commit first to update the setup.py version.